src/lexer.rs

```diff
@@ -427,30 +404,27 @@ impl<'a> Lexer<'a> {
         }
 
         if is_float {
-            // TODO: Depending on the GLSL version make it an error to not have the suffix.
-            let width = self.parse_float_width_suffix()?;
-
-            Ok(TokenValue::Float(Float {
-                value: raw
-                    .parse::<f32>()
-                    .map_err(|_| PreprocessorError::FloatParsingError)?,
-                width,
-            }))
+            // Parse the float width suffix
+            if let Some(c @ ('f' | 'F' | 'l' | 'L' | 'h' | 'H')) = self.inner.peek_char() {
```

Comment:
I know that this was already how it was done before but glsl doesn't define a `L`suffix only a `LF`

Reply:
Opened #49 for this.